client/tests/webdriver/baseSpecs/assessmentsForTasks.spec.js

```diff
@@ -87,6 +87,21 @@ describe("For the periodic task assessments", () => {
         "monthly",
         ADVISOR_1_TASK_EDIT_DETAILS
       )
+    })
+
+    it("Should allow seeing add button for the assessments in the future", async() => {
```

Comment:
The test description should probably also mention the assessment key for which you are testing this.